keps/sig-release/3031-signing-release-artifacts/kep.yaml

```diff
@@ -34,14 +34,16 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.23"
+latest-milestone: "v1.24"
```

Comment:
Are we moving the alpha milestone to 1.24?

Reply:
I think so, since it’s the latest milestone. No?

Reply:
Hehe sorry I slipped there for a second :upside_down_face: